docker/deployment/app-deployment.yaml

```diff
@@ -178,3 +178,37 @@ spec:
               secretKeyRef:
                 key: LD_TRUST_TOKEN
                 name: ld-trust-token
+      - name: lfmerge
+        # TODO: Decide on a verion-numbering policy for LfMerge releases
```

Comment:
Finally got back to this. I've updated the LfMerge build process so that it tags the Docker images with a version number that increments with every push to the `live` branch (the LfMerge branch scheme is currently `master` / `live` where Language Forge uses `develop` / `master`; I'll rename those branches soon, but haven't done so yet). The LfMerge build process also tags the Git repo with the same tags, so that it will be easy to tell which commit the Docker image was built from.

With that in place, I can finally get rid of this TODO. I've changed the lfmerge image tag in the Kubernetes deployment to lock it to a specific version. When we want to deploy a new lfmerge version, we'll just need to edit the Kubernetes deployment file and re-run `kubectl apply`. At some point I'll put that into the Makefile, but I haven't yet reached a decision on how to handle the Makefile commands (e.g. whether to use an `LFMERGE_VERSION` env var or just edit the Kubernetes file each time, and if we use the env var then how should it be calculated, that sort of thing).

But now this PR can move forward.

Reply:
Done in commit dec2d08.